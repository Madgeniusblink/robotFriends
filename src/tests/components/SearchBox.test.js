import { shallow, mount, render } from 'enzyme';
import React from 'react'
import SearchBox from '../../components/SearchBox'
import '../setupTest'


it('expect to render SearchBox component ', () => {
    expect(shallow(<SearchBox />)).toMatchSnapshot()
})

it('test input SearchBox ', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John Snow',
            username: 'JohnLuc',
            email: 'john@gmail.com'
        }
    ]
    const searchChange = mockRobots.name
    
    expect(shallow(<SearchBox onChange={searchChange} />)).toMatchSnapshot()
})