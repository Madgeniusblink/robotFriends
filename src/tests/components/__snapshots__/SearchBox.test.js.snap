// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SearchBox component  1`] = `ShallowWrapper {}`;

exports[`test input SearchBox  1`] = `ShallowWrapper {}`;
